src/harness/sas_testcase.py

```diff
@@ -96,13 +96,42 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
       self.assertEqual(grant_resp['response']['responseCode'], 0)
 
       list_of_cbsd_ids_and_grant_ids.append({
-          'cbsdId': grant_resp['cbsdId'],
-          'grantId': grant_resp['grantId']
+        'cbsdId': grant_resp['cbsdId'],
+        'grantId': grant_resp['grantId']
       })
 
     # Return cbsd_ids and grant_ids
     return list_of_cbsd_ids_and_grant_ids
 
+  def assertHeartbeatsSuccessful(self, cbsd_ids, grant_ids, operation_states):
```

Comment:
Sorry, which of the two options are you suggesting?
Option 1: return ( [cbsd1, cbsd2], [grant1, grant2] )
Option 2: return ( (cbsd1, grant1), (cbsd2, grant2) )

I think both are fine, I just prefer consistency across all functions so that they can be easily strung together. I think both can easily be used by the code that calls these functions, although option 1 is slightly closer to what we've been doing (but not necessarily better otherwise).

If you prefer option 2, consider namedtuple.